app/src/main/java/com/lorenzovainigli/foodexpirationdates/ui/theme/Theme.kt

```diff
@@ -89,14 +92,14 @@ fun FoodExpirationDatesTheme(
         else -> LightColors
     }
 
-    val systemUiController = rememberSystemUiController()
+    //val systemUiController = rememberSystemUiController()
     //val useDarkIcons = isSystemInDarkTheme()
 
-    SideEffect {
-        systemUiController.setSystemBarsColor(
-            color = colorScheme.surface
-        )
-    }
+//    SideEffect {
+//        systemUiController.setSystemBarsColor(
+//            color = colorScheme.surface
+//        )
+//    }
```

Comment:
You can directly remove these lines instead of commenting it.
If I will need to retrieve them in the future, I can look on git history.